cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -70,11 +70,23 @@ func ValidateNodeConfiguration(c *kubeadm.NodeConfiguration) field.ErrorList {
 	return allErrs
 }
 
-func ValidateAuthorizationMode(authzMode string, fldPath *field.Path) field.ErrorList {
-	if !authzmodes.IsValidAuthorizationMode(authzMode) {
-		return field.ErrorList{field.Invalid(fldPath, authzMode, "invalid authorization mode")}
+func ValidateAuthorizationModes(authzModes []string, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	for _, authzMode := range authzModes {
+		if !authzmodes.IsValidAuthorizationMode(authzMode) {
+			allErrs = append(allErrs, field.Invalid(fldPath, authzMode, "invalid authorization mode"))
+		}
+	}
+
+	found := make(map[string]bool)
```

Comment:
I'd prefer if you moved this map above the first for loop to avoid having two
Also we use `map[string]bool{}` most often instead of make, but I have no super strong opinions